@xen-orchestra/lite/src/components/Select.tsx

```diff
@@ -44,10 +46,12 @@ const Select = withState<State, Props, Effects, Computed, ParentState, ParentEff
         options?.map(item => {
           const label =
             optionRenderer === undefined
-              ? item.name ?? item.label ?? item.name_label
+              ? item.name ?? item.label ?? item.name_label ?? item
```

Comment:
In case `item` is an object, what do you think of returning `'Unknown'` ?

Reply:
1. I think it's ok if the computed *label* is `undefined`. So if `item` is a string, take that, otherwise, try `name`, `label` and `name_label` and keep whatever you get, without a warning.
2. I think it's NOT ok if the computed *value* is `undefined`. So do the same thing (but with `value`, `id`, ...) and if the final value is `undefined`, then show an error message.